modules/ROOT/pages/cloud/auth/auth.adoc

```diff
@@ -0,0 +1,7 @@
+:page-layout: api-partial
+
+. If <<use-the-api-reference-to-interact-with-the-cloud-api,issuing requests to the Cloud API from this page>> using the *Try* buttons, click the *Get Token* button. Do not edit the input field next to the button.
+. If successful, the text “1 API key applied” displays under the *Authentication* header. The token is valid for one hour.
+. Make sure to choose the correct <<servers,API server>> before making requests.
+
+See xref:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[Cloud API Authentication] for more information.
```

Comment:
If you're linking back to Redpanda docs, you need the `ROOT` (component part of the xref) because the API docs are part of a separate `api` component.

https://docs.antora.org/antora/latest/page/resource-id-coordinates/